todo_app/app.py

```diff
@@ -1,11 +1,16 @@
+from re import template
 from flask import Flask
+from flask import render_template
+
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
```

Comment:
We need a new route which accepts POST requests to create new to-do items. The URL for the route should be used in the HTML form element I have asked for below.